code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -15,6 +15,10 @@
 	var/list/obj/item/organ/internal/organs = list()
 	///Same as [above][/mob/living/carbon/var/organs], but stores "slot ID" - "organ" pairs for easy access.
 	var/list/organs_slot = list()
+	///List of [/obj/item/organ/external] in the mob, similarly used as internal_organs.
+	var/list/obj/item/organ/external/external_organs = list()
+	///Same as [above][/mob/living/carbon/var/external_organs], but stores "ID" = "organ" pairs.
+	var/list/external_organs_slot = list()
```

Comment:
mark these as orbstation changes in the comments somewhere

Reply:
agh I had commented these and my pc powered off and I forgot to do this again